core/common/assets/js/views/modal/header.js

```diff
@@ -35,5 +35,9 @@ export default class extends Marionette.LayoutView {
 
 	onCloseModalClick() {
 		this._parent._parent._parent.hideModal();
+
+		if ( elementor.config?.admin_conversion_center_url && 'links-page' === elementor.config.document?.type ) {
```

Comment:
@louiswol94 not the cleanest approach, let me what you think

Reply:
@louiswol94 no there is no command that I can hook into. How do I override a template?

Reply:
@nicoladj77 In general, I see we can open the Library with a command `library/open`, so I am not sure why we don't close it with a command as well. Then we could have used a UI hook for this. But for now, maybe you can listen for this CustomEvent `elementor/routes/close` that is dispatched and perform the redirect?

Reply:
@nicoladj77 So maybe something like this? (until we can create a proper close command).

<img width="861" alt="Screenshot 2024-05-15 at 3 05 14 PM" src="https://github.com/elementor/elementor/assets/20996103/d612b76d-6c50-4600-8090-d28e8ede7af4">

Reply:
@louiswol94 are you sure that this event is triggered?this hooks into the router but from what I see here we are just calling

	hideModal() {
		this.getModal().hide();
	}

which is in `elementor/core/common/assets/js/views/modal/layout.js`
but that to me looks like a jQuery method. 
I might be wrong but the above doesn't work and nothing is triggered from my tests.

Reply:
Hi @nicoladj77 , it was working in my above screenshot with the Links Pages. I could not test with the Conversion Center, but I assumed it's the same.

Reply:
@louiswol94 so you are right, I don't know what I was doing wrong. The issue though is that the code you suggest is triggered every time the modal is closed, including when you insert something from the library. And given it's a custom event, I could not even try to use the target to "target" it somehow.